sdk/src/main/java/io/horizen/account/api/rpc/types/EthereumTransactionView.java

```diff
@@ -79,5 +79,38 @@ public EthereumTransactionView(EthereumTransaction tx, EthereumReceipt receipt,
         blockNumber = BigInteger.valueOf(receipt.blockNumber());
         transactionIndex = BigInteger.valueOf(receipt.transactionIndex());
     }
+
+    // constructor used for the pending transactions
+    public EthereumTransactionView(EthereumTransaction tx) {
+        blockHash = Hash.ZERO;
```

Comment:
are you sure that blockHash for mempool tx must be `Hash.ZERO`, but not `null` as for other fields?
I see in eth docs `0000.0000` result, but in other docs it is `null`.
Moreover. in Geth it seems to return `null`:
```
      if blockHash != (common.Hash{}) {
		result.BlockHash = &blockHash
		result.BlockNumber = (*hexutil.Big)(new(big.Int).SetUint64(blockNumber))
		result.TransactionIndex = (*hexutil.Uint64)(&index)
	}
```